flang/include/flang/Evaluate/variable.h

```diff
@@ -160,10 +160,17 @@ class Triplet {
       std::optional<Expr<SubscriptInteger>> &&);
 
   std::optional<Expr<SubscriptInteger>> lower() const;
+  const Expr<SubscriptInteger> *GetLower() const {
```

Comment:
Looks like the case style is different than for the other function.

Reply:
Right. The front end style document says functions should begin with a capital letter with various exceptions. I flipped a coin. It can be changed.